### ISSUE TITLE
SkipNormalization not working

### ISSUE THREAD

**Issue by jcamilomnavia** [swagger-api/swagger-js#2140]
- OS: macOS
 - Environment: Chrome 91.0.4472.114, Node.js v12.6.1
 - Method of installation: yarn
 - Swagger-Client version: ^3.13.5
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration

Swagger/OpenAPI definition:
```yaml
---
openapi: 3.0.0
info:
  title: Cloudpotato - Medwork
  description: The Cloudpotato API
  version: 1.0.3
  contact: {}
tags: []
servers: []
paths:
  "/v1/clients/{id}/groups":
    get:
      operationId: getGroups
      summary: ''
      parameters:
      - name: options
        required: false
        in: query
        schema:
          type: string
      - name: id
        required: true
        in: path
        schema:
          type: number
      responses:
        '200':
          description: ''
      tags:
      - clients
      security:
      - bearer: []
  "/v1/groups":
    get:
      operationId: getGroups
      summary: ''
      parameters: []
      responses:
        '200':
          description: ''
      tags:
      - groups
      security:
      - bearer: []
```

Swagger-Client usage:
```js
const client = await new Swagger({
    url: REACT_APP_URL_SPEC,
    usePromise: true,
    requestInterceptor: SDK._requestInteceptor,
    skipNormalization: true,
});
```

### Describe the bug you're encountering
The skipNormalization property is suppose to skip the normalization for the duplicated tags that comes in the spec. This is not working as it should.

### To reproduce...

1. Get a spec with duplicated tags. For example the one above.
2. Set the skipNormalization property.

### Expected behavior
The const declared above, should have a property like:
```
client.api.client.getGroups
and
client.api.group.getGroups
```
However these are the props:
```
client.api.client.getGroups
and 
client.api.group.getGroups1
```

**Comment by char0n** [comment]
@jcamilomnavia thanks for reporting this. I'll try to reproduce this given your fixtures and get back to you.

**Comment by char0n** [comment]
I've provided the bugfix for this in https://github.com/swagger-api/swagger-js/pull/2142

The `skipNormalization` was not passed into resolver during `Swagger` class constructor instantiation.

@jcamilomnavia please check the changeset of this PR to see the actual and expected values. The expected and actual values you provided doesn't correlate with how Swagger class creates tag interfaces. It's probably just typos in your expected values, but please check the PR and tests to confirm that Swagger constructor now does you'd expect.

**Comment by jcamilomnavia** [comment]
@char0n this works just as expected. Is perfect. Thank you so much!

**Comment by char0n** [comment]
Fine, let's go for a merge then.

**Comment by char0n** [comment]
Released in 3.13.7
